src/emergency_speakup.cpp

```diff
@@ -60,6 +45,25 @@ int main(int argc, char **argv)
 	
 	ros::Rate loop_rate(1.0);
 
+    sentences_pressed.push_back("Let's pause here for a while");
+    sentences_pressed.push_back("Somebody touched a red button");
+    sentences_pressed.push_back("I may seem a little slow, but I had a rough night");
+    sentences_pressed.push_back("I am sorry for my behaviour");
+    sentences_pressed.push_back("I am a bad Amigo");
+    sentences_pressed.push_back("Stop. Hammertime!");
+    sentences_pressed.push_back("I am getting too old for this shit!");
+    sentences_pressed.push_back("Houston,,,, We have a problem");
+    sentences_pressed.push_back("I find your lack of faith disturbing");
+    sentences_pressed.push_back("Man is a robot with defects");
+
+    sentences_released.push_back("Let's move on");
+    sentences_released.push_back("Here we go again");
+    sentences_released.push_back("I'll try to do better this time");
+    sentences_released.push_back("My emergency button is released again");
+    sentences_released.push_back("Lets make this fun");
+    sentences_released.push_back("Lets get ready to rumble");
+    sentences_released.push_back("You will respect my autonomy!");
+
```

Comment:
space vs tabs issue?

Reply:
These are two different queues, so no issue at all

Reply:
I mean that the sentences are indented with 4 spaces while the rest of the code uses tabs

Reply:
Now I see the issue. I see that also the old code wasn't consistent. I will create a new PR to fix this.